### PR TITLE
DEVDOCS-2175 WordPress Install Update

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2175](https://jira.bigcommerce.com/browse/DEVDOCS-2175)

## What changed?
* updated system requirements
* updated formatting per style guide

### CODE REVIEW

File: docs/stencil-docs/developing-further/catalog-price-object.md [hunk 1]
```diff
@@ -2,76 +2,48 @@
...
+### No sale price defined in control panel
 
-#### Product in Control Panel (no Sale Price defined)
-![#### Product in Control Panel (no Sale Price defined)
-](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/sale_price_catalog.png "#### Product in Control Panel (no Sale Price defined)
-")
+Assume that a merchant has defined the following for a product in the control panel:
 
-A corresponding [Catalog Price Object](/stencil-docs/stencil-object-model-reference/stencil-objects/common-objects/catalog-price) for the product will be structured as follows:
+* **Default Price**
```
Comment (traci707) [inline docs/stencil-docs/developing-further/catalog-price-object.md]:
```suggestion
* **Default price**

```

File: docs/stencil-docs/developing-further/catalog-price-object.md [hunk 2]
```diff
@@ -379,73 +336,30 @@ lineNumbers: true
...
+ Cost |Cost  is not returned in the catalog price object.| This field represents true cost of goods, and is typically meant for reports and third-party accounting integrations. |
+MSRP (excluding tax)|`{{ product.price.rrp_without_tax }}`| This field represents the product’s list price, or the manufacturer's suggested retail price. |
+MSRP (including tax)|`{{ product.price.rrp_with_tax }}`| This field represents the product’s list price, or the manufacturer's suggested retail price, including tax. |
+Sale Price (excluding tax)|`{{product.price.sale_price_without_tax}}`| This product’s discounted/sale price. |
+Sale Price (including tax)|`{{ product.sale_price_with_tax }}`| This product’s discounted/sale price, with tax. |
+Difference between Default Price and MSRP (*Default Price - MSRP)*|`{{ product.price.saved }}`| The customer’s savings on the effective price versus list price. |
 
 ## Resources
 
 ### Related Articles
```
Comment (traci707) [inline docs/stencil-docs/developing-further/catalog-price-object.md]: Related articles

**Review by traci707 — CHANGES_REQUESTED** [review]

**Review by traci707 — APPROVED** [review]

**Review by traci707 — COMMENTED** [review]
I think all the feedback has been given.

File: docs/stencil-docs/page-builder/configuration.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+# Configuration
+
+<div class="otp" id="no-index">
+
+### On This Page
```
Comment (traci707) [inline docs/stencil-docs/page-builder/configuration.md]:
```suggestion
### On this page
```

**Review by traci707 — COMMENTED** [review]
This page was already reviewed.

**Review by traci707 — APPROVED** [review]

**Review by traci707 — APPROVED** [review]

**Review by traci707 — APPROVED** [review]

File: docs/stencil-docs/page-builder/configuration.md [hunk 4]
```diff
@@ -17,44 +17,42 @@
...
+For documentation on the principal keys included in the Cornerstone theme, see Cornerstone's [config.json Metadata](https://github.com/bigcommerce/cornerstone/blob/master/config.json) and [schema.json Metadata](https://github.com/bigcommerce/cornerstone/blob/master/schema.json) entries. 
 
 ### File management requirements
 
-* For any theme setting (such as a Sass variable or a front-matter value) to be merchant-customizable, that setting (and it's values) must be present in `schema.json`. You must manually provide these declarations, according to the structure described here.
-* Also, each key that you create in `schema.json` must have a corresponding `config.json` key whose name matches its id value. This `config.json` key sets the default value (even if that is an empty string). A `schema.json` setting without an `id`-matched `config.json` key will not appear to users in the Page Builder GUI.
+* For any theme setting, such as a Sass variable or a front-matter value, to be merchant-customizable, that setting and it's values must be present in `schema.json`. You have to manually provide these declarations, according to the structure described here.
```
Comment (traci707) [inline docs/stencil-docs/page-builder/configuration.md]:
```suggestion
* For any theme setting, such as a Sass variable or a front-matter value, to be merchant-customizable, that setting and its values must be present in `schema.json`. You have to manually provide these declarations, according to the structure described here.
```

File: docs/stencil-docs/page-builder/configuration.md [hunk 5]
```diff
@@ -17,44 +17,42 @@
...
+* Each key that you create in `schema.json` must have a corresponding `config.json` key whose name matches its id value. This `config.json` key sets the default value even if that is an empty string. A `schema.json` setting without an `id`-matched `config.json` key will not appear to users in the Page Builder UI.
```
Comment (traci707) [inline docs/stencil-docs/page-builder/configuration.md]:
```suggestion
* Each key that you create in `schema.json` must have a corresponding `config.json` key whose name matches its ID value. This `config.json` key sets the default value even if that is an empty string. A `schema.json` setting without an `id`-matched `config.json` key will not appear to users in the Page Builder UI.
```

**Review by traci707 — CHANGES_REQUESTED** [review]

**Review by traci707 — APPROVED** [review]

File: docs/stencil-docs/page-builder/configuration.md [hunk 6]
```diff
@@ -92,54 +92,54 @@ Within `schema.json`, you will also see `"type": "heading"` statements like the
...
 
 | Action | Recommendation |
 | ----------- | ----------- | 
-| Adding a new key | Adding new keys is generally fine. However, each key in `schema.json` must have a matching default in `config.json`. |
+| Adding a new key | Adding new keys is generally acceptable. However, each key in `schema.json` must have a matching default in `config.json`. |
  Deleting a key | Use caution when deleting a key. Doing so can break your new theme version's backward compatibility. | 
- Renaming a key | We do not recommend renaming keys. Instead, we recommend introducing a new key, while maintaining the old key until it is no longer in use by anyone using an older version of your theme.|
- Removing a variation| Each object within your `config.json` file's `variations` object defines one theme variation. If you are adapting an existing theme and consciously want to remove one or more variations, you can do so by removing the corresponding key(s). |
+ Renaming a key | We do not recommend renaming keys. Instead, we recommend introducing a new key, while maintaining the old key until no one using an older version of your theme uses it.|
```
Comment (traci707) [inline docs/stencil-docs/page-builder/configuration.md]:
```suggestion
 Renaming a key | We do not recommend renaming keys. Instead, we recommend introducing a new key, while maintaining the old key until no one with an older version of your theme uses it.|
```

**Review by traci707 — CHANGES_REQUESTED** [review]

**Review by traci707 — COMMENTED** [review]
Either this definition or the new one is okay for Renaming a key

**Review by traci707 — APPROVED** [review]

File: docs/api-docs/storefront/graphql/graphql-api-examples.md [hunk 7]
```diff
@@ -4,24 +4,22 @@
 
 ### On this Page
```
Comment (traci707) [inline docs/api-docs/storefront/graphql/graphql-api-examples.md]: On this page

File: docs/stencil-docs/developing-further/theme-updates-and-version-control.md [hunk 8]
```diff
@@ -3,24 +3,21 @@
 <div class="otp" id="no-index">
 
 ### On This Page
```
Comment (traci707) [inline docs/stencil-docs/developing-further/theme-updates-and-version-control.md]: On this page

File: docs/stencil-docs/localization/localizing-stores.md [hunk 9]
```diff
@@ -55,91 +49,85 @@ Localizing your theme's checkout and order confirmation pages requires customiza
...
-* [Currency-Conversion Options](/stencil-docs/template-files/customize-stencil-checkout/currency-conversion)
-* [Multi-Language Checkout](/stencil-docs/template-files/customize-stencil-checkout/multi-language-checkout)
-* [Adding Products](https://support.bigcommerce.com/s/article/Adding-Products) (BigCommerce Knowledge Base)
+* [Translation Keys](https://developer.bigcommerce.com/stencil-docs/localization/translation-keys)
+* [Restyle Optimized One-Page Checkout](https://developer.bigcommerce.com/stencil-docs/customizing-checkout/optimized-one-page-checkout)
+* [Multi-Language Checkout](https://developer.bigcommerce.com/stencil-docs/localization/multi-language-checkout)
+* [Adding Products](https://support.bigcommerce.com/s/article/Adding-Products-v3) (BigCommerce Knowledge Base)
 * [Changing Your Default Currency](https://support.bigcommerce.com/s/article/Managing-Currencies#default)(BigCommerce Knowledge Base)
 
 ### Additional Resources
```
Comment (traci707) [inline docs/stencil-docs/localization/localizing-stores.md]: Additional resources

**Review by traci707 — CHANGES_REQUESTED** [review]

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 10]
```diff
@@ -4,30 +4,30 @@
...
+### On this page
+- [System requirements:](#system-requirements)
 - [Installation](#installation)
-- [Setting Your Channel Name](#setting-your-channel-name)
+- [Setting your channel name](#setting-your-channel-name)
 
 </div>
 
-## System Requirements:
+## System requirements:
```
Comment (uwikaiddi) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:14]:
```suggestion
## System requirements

```

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**